README.rst

```diff
@@ -1,3 +1,18 @@
+.. image:: https://img.shields.io/pypi/v/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+
+.. image:: https://img.shields.io/pypi/l/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+
+.. image:: https://img.shields.io/pypi/wheel/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+
+.. image:: https://img.shields.io/pypi/pyversions/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+
+.. image:: https://travis-ci.org/Pylons/waitress.svg?branch=master
+    :target: https://travis-ci.org/Pylons/waitress
+
```

Comment:
When new branches are released, there will be problems with the PyPI badges as it does not support branches. That will provide incorrect information. So I'm a -1 on all the PyPI badges.

To help clarify for docs authors what these badges are, please add an `:alt:` option. See [example from Pyramid](https://github.com/Pylons/pyramid/blame/master/README.rst#L4-L18).

I'd also like to standardize badges across Pylons Projects:
* Docs for this branch only
* Build status for this branch (Travis-CI)
* IRC